Description: Fix Debian bug #700680
Author: Mikolaj Izdebski <zurgunt@gmail.com>
Bug-Debian: http://bugs.debian.org/700680
Applied-Upstream: commit: 7d85774c8bae5935101097ff5ba621a4eafcf8a8
Last-Update: 2013-03-02
--- a/src/main.c
+++ b/src/main.c
@@ -702,7 +702,7 @@ input_init(const struct arg *operand, struct stat *sbuf)
       return -1;
     }
 
-    if (!S_ISREG(sbuf->st_mode)) {
+    if (OM_REGF == outmode && !S_ISREG(sbuf->st_mode)) {
       warn("skipping \"%s\": not a regular file", operand->val);
       return -1;
     }
